src/components/client/core/Button/Button.stories.tsx

```diff
@@ -1,4 +1,4 @@
-import Button from "./Button";
+import { Button } from "components/client";
```

Comment:
Good call on the aliased imports, I love doing this for portability/easier refactors